src/app/components/CallToActionLinkComponent/Text/index.tsx

```diff
@@ -0,0 +1,40 @@
+/** @jsx jsx */
+
+import { PropsWithChildren, useContext } from 'react';
+import { jsx } from '@emotion/react';
+import TEXT from '#app/components/Text';
+import CallToActionLinkContext from '../CallToActionLinkContext';
+import styles from './index.styles';
+
+type TextProps = {
+  as?: string;
+  className?: string;
+  shouldUnderlineOnHoverFocus?: boolean;
+};
+
+const Text = ({
+  as,
+  children,
+  className,
+  shouldUnderlineOnHoverFocus,
+}: PropsWithChildren<TextProps>) => {
+  const { fontVariant, size } = useContext(CallToActionLinkContext);
+  return (
+    <TEXT
+      as={as}
+      size={size || 'pica'}
+      fontVariant={fontVariant || 'sansBold'}
```

Comment:
Do we need to include the defaults here, if they have already been defaulted in the CallToActionLinkContext?

```suggestion
      size={size}
      fontVariant={fontVariant}
```

Reply:
Good point, I've defaulted them as props in the CallToActionLink. (Wasn't sure how to do this specifically within the context)